docs/articles/oso/how-to-open-source-software.rst

```diff
@@ -0,0 +1,28 @@
+:orphan:
+
+How to open source software
+===========================
+
+.. tagslist:: howto
+
+When open sourcing source code there are some steps that needs to be completed.
+The purpose of these steps are to ensure that the code has a sound license, the
+source code repository is understandable and that there are clear descriptions
+on how to contribute. Following is a listing of these steps:
+
+#. Make sure that the following files are present in the root of the source code
+   repository:
+    * LICENSE: Containing the desired license for the code
+    * README.md: Containing information on where to find the documentation and
+      simple getting started info.
+    * CONTRIBUTING.md: Containing information on how to contribute to the
+      project and what rules there are.
+#. Go through file headers of all the source code and make sure that there is a
+   reference to the LICENSE file and an SPDX tag.
+#. Send a request to open source the code to the Open Source Office (OSO). The
```

Comment:
I think the context here is missing. It cannot be obvious for a reader what is an "Open Source Office". Is it some well known institution that everyone can refer to? 

It is the community manager in your company or the description on what it is should exist.

Reply:
I agree this is missing, but that might be difficult to add as part of this story. Unless we do a complete placeholder with a very brief description.

It's part of the current backlog in  a project to add a description of OSO and the roles and activities around that.